lib/ansiblelint/skip_utils.py

```diff
@@ -0,0 +1,163 @@
+# (c) 2012-2019, Ansible by Red Hat
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+"""This module contains utils related to inline skiping of rules."""
+
+from itertools import product
+
+import ruamel.yaml
+
+INLINE_SKIP_FLAG = '# noqa'
+
+
+def get_rule_skips_from_line(line):
+    """Return list of rule ids skipped via comment on the line of yaml."""
+    rule_id_list = []
+    if INLINE_SKIP_FLAG in line:
+        noqa_text = line.split(INLINE_SKIP_FLAG)[1]
```

Comment:
It'd be probably cleaner to replace the whole function body with just:
```suggestion
    _before_noqa, _noqa_marker, noqa_text = line.partition(INLINE_SKIP_FLAG)
    return noqa_text.split()
```
(though, it can wait till the next refactoring iteration)